evostar2019/angrybirds-evo19.tex

```diff
@@ -122,8 +124,40 @@
 %
 \section{Introduction}
 \label{sec:intro}
-\input{Context}
-\input{Motivation}
+%\input{Context}
+%\input{Motivation}
+
+% Antonio - I have written a text closer to what a paper needs to include. We must be quite concise and explain just what it is required to justify our work. ;D
+% Moreover, for a document as short as a paper is, I would suggest to work on a single TEX file. 
+
+One of the main challenges nowadays in the videogames development is the automatic generation of contents, as games are growing in complexity exponentially \cite{togelius2016introduction}. Thus, there is a vast amount of resources and assets to be created by the artistic staff, designers and programmers, who would profit some tools to automatise or simplify their tasks.
+
+Procedural Content Generation (PCG) brings a possible solution to this situation. It is defined as \textit{the algorithmic creation of game content with limited or indirect user input} \cite{togelius2011procedural}. 
+PCG often uses Artificial Intelligence (AI) techniques and normally includes aesthetic elements, game rules, levels, items, stories and character designs among others contents or assets.
+
+The main objective to reach for a videogame is the \textit{player's engagement}, which is mainly based on two factors: replayability (how interesting is playing a game more than once) and adaptability (adapting gameplay elements to each individual player). Replay value strongly depends on the offer of new (and attractive) content for the player, once he/she has finished the main game. The second factor normally lies on the possibility of configuring some game aspects in a menu, such as the game difficulty, character control, or camera options. However, more advanced approaches implement adjustments based on in-game player behaviour: automatically regulating its difficulty level to fit the player's learning curve or creating content that matches the player's taste.
+These are also PCG applications.
+
+% Antonio - a paragraph saying which of these factors (replayability or adaptability), or both, are we addressing in this paper.
+
```

Comment:
If you want to do that, work on the existing introduction, which had undergo two rounds of review already.